acts

```diff
@@ -161,6 +146,20 @@ backuprc=0 # Notice any failed backups
 for dir in $backuptargets; do
```

Comment:
Can you add a log_debug message specifying which backup target is being processed at the start of the loop?